service/awsconfig/v6/version_bundle.go

```diff
@@ -0,0 +1,55 @@
+package v6
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
+	return versionbundle.Bundle{
+		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "aws-operator",
+				Description: "There are no changes yet, replace this with something useful.",
+				Kind:        versionbundle.KindChanged,
+			},
```

Comment:
This is the new WIP one. Just some mock info in here which we have to fill with real info at some point. I was thinking to improve the validation of the version bundles because we require changelogs, which does not make that much sense for WIP bundles in all cases. Uncertain.